src/DotNetLightning.Core/Channel/Channel.fs

```diff
@@ -54,12 +62,20 @@ module Channel =
         |> fun ecdsaSig -> TransactionSignature(ecdsaSig.Value, SigHash.All)
 
     module Closing =
-        let makeClosingTx (keyRepo: IKeysRepository, cm: Commitments, localSpk: Script, remoteSpk: Script, closingFee: Money, localFundingPk, n) =
+        let makeClosingTx (
+                channelPrivKeys: ChannelPrivKeys,
+                cm: Commitments,
```

Comment:
I think you're introducing indentation warnings here

Reply:
Fixed.